enterprise/metrics/pom.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.neo4j</groupId>
+    <artifactId>parent</artifactId>
+    <version>2.3-SNAPSHOT</version>
+    <relativePath>../..</relativePath>
+  </parent>
+  
+  <artifactId>neo4j-metrics</artifactId>
```

Comment:
yes that i know it's really hard to read that in case of you pom is to big even if IDEA suggest to remove it.
The only field that can be removed here is version field because that is inherited and submodule will never gonna have version different from a parent. That is a common pattern to show that component is part of something bigger.